features/support/run_helpers.rb

```diff
@@ -63,12 +63,12 @@ def result_has_shell_error? result
 end
 
 
-def run_shell_command command, input
+def run_shell_command command, inputs
   result = OpenStruct.new(command: command, location: Pathname.new(Dir.pwd).basename)
   command = "#{shell_overrides}; #{command} 2>&1"
 
   status = Open4.popen4(command) do |_pid, stdin, stdout, _stderr|
-    stdin.puts input if input
+    inputs.each { |input| stdin.puts input }
```

Comment:
Let's rename `inputs` to `input` and make it a [casual API](http://blog.originate.com/blog/2014/06/26/casual-apis/).
* if a string is given, use the string
* if an array is given, use each string inside the array